nexus_constructor/model/transformation.py

```diff
@@ -84,6 +84,14 @@ def units(self):
     def units(self, new_units):
         self.set_attribute_value(CommonAttrs.UNITS, new_units)
 
+    @property
+    def value(self) -> Dataset:
```

Comment:
i think instead of using a property here we should use the `values` member of the dataset, using an extra attribute doesn't match the JSON. unless you think it is easier to use a property?

Reply:
I think we should copy the previous `UI_value` functionality as well in the old transformations class (ui value is dataset if scalar, float/int castable, else use the ui value stored as an attribute)